protocols/bee.h

```diff
@@ -58,10 +58,11 @@ void bee_free( bee_t *b );
 /* TODO(wilmer): Kill at least the OPT_ flags that have an equivalent here. */
 typedef enum
 {
-	BEE_USER_ONLINE = 1,    /* Compatibility with old OPT_LOGGED_IN flag */
-	BEE_USER_AWAY = 4,      /* Compatibility with old OPT_AWAY flag */
-	BEE_USER_MOBILE = 8,    /* Compatibility with old OPT_MOBILE flag */
-	BEE_USER_LOCAL = 256,   /* Locally-added contacts (not in real contact list) */
+	BEE_USER_ONLINE = 1,       /* Compatibility with old OPT_LOGGED_IN flag */
+	BEE_USER_AWAY = 4,         /* Compatibility with old OPT_AWAY flag */
+	BEE_USER_MOBILE = 8,       /* Compatibility with old OPT_MOBILE flag */
+	BEE_USER_LOCAL = 256,      /* Locally-added contacts (not in real contact list) */
+	BEE_USER_SPECIAL = 262144, /* Compatibility with old OPT_SPECIAL flag */
```

Comment:
@Wilm0r is there any meaning behind these values?  I understand the compatibility flags, however, `BEE_USER_LOCAL` jumps straight to 256 (`0x100`), which is `OPT_TYPING`.  So, it would appear they're not completely "compatible."  Are any unused values up for grabs?